src/extension/app.tsx

```diff
@@ -51,6 +51,7 @@ chrome.runtime.sendMessage({ fetch: true }, function (response) {
             if (!(id in structuredEvents)) {
                 structuredEvents[id] = { envelope: json.envelope, events: [] };
                 structuredEvents[id]["envelope"].dateTime = entry.dateTime;
+                //structuredEvents[id]["envelope"].PLTV2 = 123; // LOGIC ;
```

Comment:
> //structuredEvents[id]["envelope"].PLTV2 = 123; // LOGIC ; [](start = 15, length = 59)

revert this file

Reply:
not fixed yet

---
In reply to: [175848321](https://github.com/Microsoft/clarity-tools/pull/41#discussion_r175848321) [](ancestors = 175848321)